zebra/zebra_evpn_mh.c

```diff
@@ -325,16 +325,26 @@ zebra_evpn_es_evi_show_entry_detail(struct vty *vty,
 	if (json_array) {
 		json_object *json;
 		json_object *json_flags;
+                json_object *json_types;
 
 		/* Separate JSON object for each es-evi entry */
 		json = json_object_new_object();
 
 		json_object_string_add(json, "esi", es_evi->es->esi_str);
 		json_object_int_add(json, "vni", es_evi->zevpn->vni);
+                if (es_evi->flags & ZEBRA_EVPNES_EVI_LOCAL) {
+                        json_types = json_object_new_array();
+                        if (es_evi->flags & ZEBRA_EVPNES_EVI_LOCAL)
+                                json_array_string_add(json_types, "local");
+                        json_object_object_add(json, "type", json_types);
```

Comment:
This sounds to me like `types`, not `type`, no?

Reply:
It is "type"
```
cumulus@torm-11:mgmt:~$ net show bgp l2vpn evpn es json 
[
  {
    "esi":"03:44:38:39:ff:ff:01:00:00:01",
    "rd":"27.0.0.15:3",
    "type":[
      "local",
      "remote"
    ],
    "vniCount":10
  },